app/views/compute_resources_vms/form/_openstack.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
```

Comment:
It works for me, and all this is doing is reverting the part of the commit that broke this, e972c0e.

We should probably take the opportunity to fix this completely which means supporting multiple security groups, but the underlying compute resource code, combined with the fog extensions, makes it hard to trace through the code exactly what is happening.

So, if anyone wants to take this and fix it, please do, otherwise I'll come back when I have time.